website/common/script/ops/buy/buyQuestGem.js

```diff
@@ -8,6 +8,7 @@ import content from '../../content/index';
 
 import errorMessage from '../../libs/errorMessage';
 import { AbstractGemItemOperation } from './abstractBuyOperation';
+// import itemInfo from '../../libs/getItemInfo';
```

Comment:
We decided against doing this, no need to add a comment for it.

Reply:
That's an artifact of a previous attempt--will delete.